sonic-net/sonic-buildimage
# Install barefoot

#### Why I did it
This makes the genl-packet kernel module available in barefoot builds
#### How I did it
Added to install list in platform/barefoot/one-(image/aboot).mk
#### How to verify it
Build barefoot image and ensure kernel module is built and loaded into the install image
#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog


#### Ensure to add label/tag for the feature raised. example - PR#2174 under sonic-utilities repo. where, Generic Config and Update feature has been labelled as GCU.

#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
